frontend/public/components/edit-yaml.jsx

```diff
@@ -264,47 +289,69 @@ export const EditYAML_ = connect(stateToProps)(
           });
         }
 
-        save() {
-          const { onSave, t } = this.props;
-          const { owner } = this.state;
-          let obj;
+        createResources(objs) {
```

Comment:
OK I see why you used `createRequests` above, but it's sort of confusing that we have two functions with slightly different names that do slightly different things.  I would try to make it clear in the name somehow the difference between the two functions.

Reply:
used createResources above and performDryRun here